charts/k8ssandra/values.yaml

```diff
@@ -97,6 +97,10 @@ cassandra:
     # environments.
     size: 1
 
+    # -- Number of tokens within the datacenter. If not defined, the default values will be
```

Comment:
I would be explicit about the defaults. Go ahead and mention that for 3.11 the default is 256 and for 4.0 the default is 16.